packages/yourloops/pages/hcp/team-members-table.tsx

```diff
@@ -51,14 +51,15 @@ import Typography from "@material-ui/core/Typography";
 
 import AccessTimeIcon from "@material-ui/icons/AccessTime";
 import ExpandMoreIcon from "@material-ui/icons/ExpandMore";
-import PersonRemoveIcon from "../../components/icons/PersonRemoveIcon";
 import IconActionButton from "../../components/buttons/icon-action";
+import PersonRemoveIcon from "../../components/icons/PersonRemoveIcon";
 
 import { UserInvitationStatus } from "../../models/generic";
 import { TeamMemberRole, TypeTeamMemberRole } from "../../models/team";
 import { getUserFirstName, getUserLastName, getUserFirstLastName, getUserInitials } from "../../lib/utils";
 import { useAuth } from "../../lib/auth";
 import { Team, TeamMember, useTeam } from "../../lib/team";
+import CertifiedProfessionalIcon from "../../components/icons/certified-professional-icon";
```

Comment:
weird, this import is not where it should be ... do you use auto import ordering in WS ?

Reply:
My bad, Webstorm hides the imports, I didn't see it !!